packages/collector/test/tracing/protocols/apollo_subgraph/client.js

```diff
@@ -1,6 +1,5 @@
 /*
- * (c) Copyright IBM Corp. 2021
```

Comment:
issue: We never change the copyright year in an existing JS file. (even if we rename the file)

Reply:
Removed.